sandbox/settings.py

```diff
@@ -119,7 +107,7 @@
 )
 
 # Make this unique, and don't share it with anybody.
-SECRET_KEY = '$)a7n&o80u!6y5t-+jrd3)3!%vh&shg$wqpjpxc!ar&p#!)n1a'
+SECRET_KEY = env.str('SECRET_KEY', default='$)a7n&o80u!6y5t-+jrd3)3!%vh&shg$wqpjpxc!ar&p#!)n1a')
```

Comment:
Seems like this line is causing an issue with the sandbox and docs suites, though, on the face of it, I can't see what the issue is.

Reply:
Apparently django-environ doesn't handle special characters in strings too well.

Reply:
A $ prefix is a special case for proxy values: https://django-environ.readthedocs.io/en/latest/#proxy-value

Reply:
Good to know, thanks!